### PR TITLE
qat: add support for 401xx devices

### PR DESCRIPTION
**Pull request by mythi**
QAT_401xx is a derivative of 4xxx. Add support for that device
by including the device IDs (both PF and VF).

Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]
/lgtm

**Comment by bart0sh** [comment]
@mythi I don't have merge rights for this PR it seems.

**Comment by mythi** [comment]
> @mythi I don't have merge rights for this PR it seems.

we have new codeowners so we need to wait @hj-johannes-lee approval too

**Review by tkatila — APPROVED** [review]
Looks good :)

**Review by hj-johannes-lee — APPROVED** [review]
LGTM :D

**Comment by mythi** [comment]
@bart0sh you should be able to merge now (but @hj-johannes-lee could have  done it too)

**Pull request merged**
